backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -0,0 +1,220 @@
+<script lang="ts">
+    import { cn } from "$functions/classname";
+    import ScrollArea from "$components/minor/ScrollArea/Index.svelte";
+    import Tick from "$icons/Tick/Index.svelte";
+    import MoreBox from "$icons/MoreBox/Index.svelte";
+
+    // Mapping
+    let filter_options_mapping: {
+        [key: string]: {
+            title: string;
+            class: string;
+            value: string;
+            items?: Record<string, string> | undefined;
+            selected_items: Array<[string, string]>;
```

Comment:
`selected_items?: Array<[string, string]>;`

Reply:
We need to initialize an array anyway, ??
`filter_option.selected_items = [...filter_option.selected_items, selected_item];`

Oh you mean just this type? dont change mapping? Okay